torch/fx/proxy.py

```diff
@@ -259,6 +275,24 @@ def node(self):
     def __call__(self, *args, **kwargs):
         return self.tracer.create_proxy('call_method', self.attr, (self.root,) + args, kwargs)
 
+
+class FixedShapeParam(Proxy):
```

Comment:
I'd recommend naming this something like `ParameterProxy` to indicate that it's a proxy value